src/components/backend-ai-registry-list.ts

```diff
@@ -51,6 +51,13 @@ class BackendAIRegistryList extends BackendAIPage {
   @property({type: String}) registryType = 'docker';
```

Comment:
Also, Since accessing to attributes of the domain is only allowed to "superadmin" role(refer [here](https://github.com/lablup/backend.ai/blob/main/src/ai/backend/manager/models/domain.py)), which should be protected from any manipulation occurred from outside of the component.
IMO, It's better to declare member variable with `@state()` and `private`.

Reply:
And I wonder whether we need to add `public` access modifier to `registryList`. If there's no specific reason to access from outside of the component, let's replace the access modifier with `private`.

Reply:
> We already defined `registryType` in L57 (`registerType` seems like typo error), but there's still `registryType` property declaration left in L51. Please remove the property unless you need to expose them in public for a certain reason.

The registryType declared as a property and the member variable registerType are used for different purposes. Therefore, I have clearly changed the name of the register type without removing either one.

Reply:
and I applied it